### PR TITLE
[Merged by Bors] - Update stacks kafka-druid-superset-s3 and trino-superset-s3 to release 22.09

### PR DESCRIPTION
**Pull request by sbernauer**
## Description

Tested stacks and demos,  worked fine

## Review Checklist
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: stacks/stacks-v1.yaml [hunk 1]
```diff
@@ -52,12 +52,12 @@ stacks:
               password: superset
               database: superset
       - plainYaml: https://raw.githubusercontent.com/stackabletech/stackablectl/main/stacks/kafka-druid-superset-s3/zookeeper.yaml
-      - plainYaml: https://raw.githubusercontent.com/stackabletech/stackablectl/main/stacks/kafka-druid-superset-s3/kafka.yaml
+      - plainYaml: stacks/kafka-druid-superset-s3/kafka.yaml # TODO
```
Comment (sbernauer) [inline stacks/stacks-v1.yaml:55]: marker

File: stacks/stacks-v1.yaml [hunk 2]
```diff
@@ -124,7 +124,7 @@ stacks:
               password: superset
               database: superset
       - plainYaml: https://raw.githubusercontent.com/stackabletech/stackablectl/main/stacks/trino-superset-s3/hive-metastore.yaml
-      - plainYaml: https://raw.githubusercontent.com/stackabletech/stackablectl/main/stacks/trino-superset-s3/trino.yaml
+      - plainYaml: stacks/trino-superset-s3/trino.yaml # TODO
```
Comment (sbernauer) [inline stacks/stacks-v1.yaml:127]: marker

**Review by maltesander — APPROVED** [review]
LGTM.

**Pull request closed without merging**